website/docs/docs/dbt-cloud/cloud-api.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "dbt Cloud API"
+id: "cloud-api"
+---
+
+### Overview
+
+If you are on the `Team` or `Enterprise` dbt Cloud plan, you will have access to the 
+dbt Cloud API.
+
+Common uses for our API would be:
+
+- Downloading artifacts after a job has completed 
+- Kicking off a job run 
+- Creating/modifying/deleting objects in dbt Cloud 
+
+Our API endpoints can be found [here](/dbt-cloud/api).
+
+### How to Access the API 
+
+#### User API tokens
+You can find your API token in your user's profile under `API Access`.  
+
+<Lightbox src="/img/api-access-profile.png" title="API Access" />
+
+#### Service API token
+To enable this for your account, you will need to contact Support via Intercom 
+or support@getdbt.com. Once it is enabled, you can create these tokens by going to 
+Account Settings > Service Tokens.
+
+<LoomVideo id="/55eb660a52bf4417b1274c6a55796cda" />
```

Comment:
I don't think this works - you might need to remove the leading slash? See the deploy preview here: https://deploy-preview-544--docs-getdbt-com.netlify.app/docs/dbt-cloud/cloud-api